code/datums/gamemode/misc_gamemode_procs.dm

```diff
@@ -342,6 +342,7 @@
 
 			T = new(R)
 			T.uses = uses
+			T.job = role.name_for_uplink
```

Comment:
instead of pasting this twice, just write it once on line 366, since that confirms an uplink is present

Reply:
done, compiled, and tested